atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -42,6 +43,7 @@ def __init__(self, tasker, workflow, remote_sources=None):
         """
         super(DownloadRemoteSourcePlugin, self).__init__(tasker, workflow)
         self.remote_sources = remote_sources
+        self.new_remote_sources = None
```

Comment:
I don't like `new_` prefix, will we have also `new_new_new_remote_sources` ? Can we rename it to something more feature related names?

Reply:
fixed